include/blmc_robots/n_joint_blmc_robot_driver.hpp

```diff
@@ -458,4 +632,30 @@ class NJointBlmcRobotDriver
     }
 };
 
+template <typename Driver>
+typename Driver::Types::BackendPtr create_backend(
+    typename Driver::Types::DataPtr robot_data,
+    const std::string &config_file_path = "")
+{
+    constexpr double MAX_ACTION_DURATION_S = 0.003;
+    constexpr double MAX_INTER_ACTION_DURATION_S = 0.005;
```

Comment:
i think we should make the two values above also parameters

Reply:
I agree. This will require some structural changes, though, so I would prefer to add this in a later PR to not delay this one too much.